### ISSUE TITLE
Add "Language" field for listings

### ISSUE THREAD

**Issue by JustinDrake** [OpenBazaar/openbazaar-desktop#57]
At Duo Search we're trying to filter listings by language. (E.g. an English might not want to see listings in German, French or Spanish.)

It would be very useful if the vendor specified the language used for his listing, so that filtering by language can be done automatically.

**Comment by hoffmabc** [comment]
I think this would be super useful as well.

**Comment by morebrownies** [comment]
Hey @jjeffryes weren't we planning on adding a language field to the Settings > Page tab? I believe we discussed pulling it out of the Moderation tab so that we could use it for other purposes too. 

If so, could the set languages also get pulled into each listing? 

![screenshot 2017-02-17 11 36 02](https://cloud.githubusercontent.com/assets/13066886/23080365/4add57de-f505-11e6-994f-6166d82963bc.png)

**Comment by rmisio** [comment]
> If so, could the set languages also get pulled into each listing?

If you mean at listing creation, I don't think that's a problem.

**Comment by morebrownies** [comment]
Well, if I updated my languages wouldn't it need to republish to each listing as well?

**Comment by rmisio** [comment]
Yeah, but just because a user changed the languages they speak doesn't mean the listing is in all the languages.

If anything, during listing creation we could pre-populate the lang field with those languages and the user could pare them down, if necessary.

Or, if it's much more likely the user is going to make a listing in only one language, then we could just leave the field empty and leave it up to user to populate.

**Comment by jjeffryes** [comment]
A language field exists in each listing in metadata.language, but it is a string, so it will only accept one value currency.

https://github.com/OpenBazaar/openbazaar-go/blob/master/pb/contracts.pb.go
